gef-ui/server/utils/helpers.js

```diff
@@ -116,46 +128,44 @@ const returnToMakerNoFacilitiesChanged = (app, hasChangedFacilities) => {
     app.submissionCount > 0
   );
 };
+/**
+ * @param {object} params
+ * @param {string} params.href - the URL to navigate to
+ * @param {string} params.visuallyHiddenText - the visually hidden label to make it clear to a screen reader what the link is changing
+ * @param {'Change' | 'Add' | undefined} params.actionType - the text to display, component has class display-none if falsy
```

Comment:
Mentioned elsewhere about pulling 'Change' and 'Add' out as constants - maybe instead make an `ActionTypes` at the top of the file (or wait, maybe that would need to be in a separate typescript file..?), and then we can use that here too?

Reply:
`{import('@ukef/dtfs2-common').ValuesOf<typeof ACTION_TYPE> | undefined}` almost works but jsdoc isnt quite smart enough to work this out :/ not sure which I prefer